tests/test_project.py

```diff
@@ -165,6 +165,8 @@ def mock_project(self):
             project.open_job(dict(i=i)).init()
         return project
 
+    @pytest.mark.skipif(signac.__version__ < '1.3.0',
+                        reason='__contains__ was refactored to run in constant time in 1.3.0 ')
```

Comment:
I think this reason needs to be a bit more detailed. It's not clear from this reason what method you're referring to or why it would be relevant (for instance, is it `StatusRenderer.__contains__`?). I would be a little more descriptive indicating which method exactly and why it's related to this test.